src/main/kotlin/com/exactpro/th2/conn/amqp/MessagePublisher.kt

```diff
@@ -54,6 +72,9 @@ class MessagePublisher(
         } catch (ex: Exception) {
             LOGGER.error(ex) { "Cannot add message for direction $direction. ${holder.body.contentToString()}" }
         }
+        counters[direction]?.run {
+            labels(sessionAlias).inc()
```

Comment:
If we have only one session alias in the current implementation then we can cache the result of the labels method call